### PR TITLE
NAS-121456 / 23.10 / Adds field for emptyDir `sizeLimit` to ix-chart

### PR DESCRIPTION
**Pull request by stavros-k**
Adds a field for sizeLimit on memory backed volumes. It's an optional field

### CODE REVIEW

File: library/ix-dev/charts/ix-chart/questions.yaml [hunk 1]
```diff
@@ -487,6 +487,14 @@ questions:
                 schema:
                   type: path
                   required: true
+              - variable: sizeLimit
```
Comment (Qubad786) [inline library/ix-dev/charts/ix-chart/questions.yaml:490]: Let's add a reasonable default?

File: library/ix-dev/charts/ix-chart/questions.yaml [hunk 2]
```diff
@@ -487,6 +487,14 @@ questions:
...
+                label: "Size Limit"
+                description: |
+                  Optional - Size of the memory backed volume.</br>
+                  Format: 100Mi, 1Gi, 2Gi etc
+                schema:
+                  type: string
+                  valid_chars: "^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$"
```
Comment (Qubad786) [inline library/ix-dev/charts/ix-chart/questions.yaml:497]: I wonder if we can somehow have unit tests for such regex matches?

File: library/ix-dev/charts/ix-chart/questions.yaml (hunk 1 above)
Comment (stavros-k) [reply library/ix-dev/charts/ix-chart/questions.yaml:490]: I think 2Gi is okay for most systems?

Comment (Qubad786) [reply library/ix-dev/charts/ix-chart/questions.yaml:490]: Maybe 1Gi?

Comment (Qubad786) [reply library/ix-dev/charts/ix-chart/questions.yaml:490]: Probably even less then that might be better..

**Review by Qubad786 — APPROVED** [review]
Please fix the conflict otherwise LGTM ✅

**Pull request merged**